src/patternfly/components/Menu/templates/menu-breadcrumbs--Drilldown.hbs

```diff
@@ -28,8 +28,17 @@
           {{/breadcrumb-link}}
         {{/breadcrumb-item}}
 
-        {{#> menu-breadcrumbs--dropdown dropdown-toggle--Badge--text="1" dropdown--IsExpanded="true"}}
-          {{> dropdown-menu-list-item dropdown-menu-item--text='<span class="pf-v5-c-dropdown__menu-item-icon"><i class="fas fa-angle-left" aria-hidden="true"></i></span>Edit'}}
+        {{#> menu-breadcrumbs--dropdown menu-toggle--badge--text="1" menu-toggle--IsExpanded=true}}
+          {{#> menu-list-item}}
+            {{#> menu-item}}
+              {{#> menu-item-main}}
+                {{> menu-item-toggle-icon}}
```

Comment:
It would be great if we can use the toggle icon here since it gives us the display we want and it seems like a match in the general expected behavior - it takes you backward. IMO it would be great if we can have the ability to specify an item as going forward/backward (or some better wording) and it adds the toggle icon. That will fix this issue, but also give us consistency in these arrows, and also free up the `icon` prop on these items that are adding the arrow using the `icon` prop.